### DISCUSSION TITLE
Welcome to Earth-Analytics-AY24 Discussions!

### DISCUSSION THREAD

**Discussion by nquarder** [earthlab-education/Earth-Analytics-AY24#3] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by gpb3037** [comment]
Just checking in - maybe confused on where to find the syllabus at https://github.com/earthlab-education/Earth-Analytics-AY24/wiki but that currently seems empty (maybe this gets populated first day of class?)

**Reply by gpb3037** [reply]
looks like the wiki is alive now.  There is of course more stuff on https://canvas.colorado.edu/courses/109233

**Reply by gpb3037** [reply]
also https://cu-esiil-edu.github.io/esiil-learning-portal/foundations/pages/00-course-overviews/foundations/01-fundamentals/00-home.html and subsequent pages report

An error occurred: giscus is not installed on this repository

**Reply by eculler** [reply]
Thanks @gpb3037, I've updated the giscus, so it should now route to this discussion board if you want to make any comments on textbook pages.

**Comment by BioNCphilic** [comment]
Greetings - I'm a newbie, but excited about learning. I work on a [NASA Science Activation team](https://science.nasa.gov/learn/science-activation-team/) and am excited to be a data translator for public school teachers who want to empower their students.

**Reply by nquarder** [reply]
Great meeting you yesterday @BioNCphilic and welcome to the program! We're excited to have you here this year 👍 -NQ

**Comment by gpb3037** [comment]
I was watching a 10minute intro video by Dr. Culler lost connection, now can't find it again - can anyone help with a link?

**Reply by nquarder** [reply]
Hi @gpb3037 ! Here is a link to @eculler's Welcome video:

https://canvas.colorado.edu/courses/109233/modules/items/5794357 (From here click the button that says 'Open Welcome video (Elsa) in a new window'

I found this by going to the course page in Canvas and then selecting Modules. It is in the module titled 'Create your Earth Data Science Portfolio' near the top. Hope this helps! -NQ

**Comment by ZimmZone** [comment]
Hi everyone. I am Erin and I am a political scientist by training and a writer by profession. I have been an environmental advocate for years and wanted to move more into evidence based advocacy. I am looking forward to learning all about working with large data sets and meeting everyone in the course.

**Reply by nquarder** [reply]
Great to have you in class this semester @ZimmZone - We look forward to hearing more about your interests and value your contributions to this course! 👍 -NQ

**Comment by npwelsh** [comment]
Great first class!  I'm Nolan, have a background in geology and 10 years experience working as downhole geophysical engineer.  Currently at CU studying renewable energy.  Looking forward to learning with everyone!

**Reply by nquarder** [reply]
Excited to have you in class this semester @npwelsh 😎

**Comment by allenmoench** [comment]
Hello! I'm Michael. I studied environmental science and sustainability at Western Washington University. I have a year and a half of experience as a Wildland Firefighter in mitigation and suppression, and recently completed a certificate program in Foundations of Geographical Information Systems through Front Range Community College. I'm currently working as a GIS analyst. I'm particularly interested in studying climate solutions, and I'm looking for graduate programs to starting next fall. Looking forward to digging into the data with you all!

**Reply by nquarder** [reply]
Welcome @allenmoench - you background and prior experiences will have you prepared for this course. Glad to have you on our team this semester! -NQ

**Reply by WillToon** [reply]
Hey Micheal, I graduated from WWU in 2020. It's nice to see another Western Alumni here! I studied Geography & Environmental Studies and now am beginning a consulting position in GIS this month. Looks like we both have similar backgrounds and looking forward to improving our coding skills!

**Reply by allenmoench** [reply]
Hi Will, great to see another WWU alumni in the class. I'm curious to hear more about what you studied, and where you're headed!

**Comment by annaposlednik** [comment]
Hi I'm Anna, and I'm a first year PhD student in Ecology and Evolutionary Biology at CU Boulder. My research focuses on marine parasites and diseases, and I'm excited to learn how to most effectively analyze large spatial datasets which I then hope to apply to my own work.

**Reply by nquarder** [reply]
Glad to have you in class @annaposlednik and looking forward to working with you this semester. -NQ

**Comment by hanried** [comment]
Hi all! My name is Hannah and I am currently a research administrator at CU Boulder. We help faculty and departments with the logistics of applying for and managing research grants/contracts. Previously, I taught high school math for a couple of years after graduating from CU. I'm excited to learn how to code and re-engage some of my math critical thinking skills now that I've had a chance to relax and reassess my career goals since leaving teaching. I'm also excited to dig into some earth data!

**Reply by nquarder** [reply]
Great having you in class this semester @reidhan - looking forward to getting to know you. -NQ

**Comment by ghost** [comment]
Hi everyone! My name is Lauren and I am currently the (only) Landmark Preservation Inspector for Denver and exclusively do construction inspections on properties that are individual landmarks or in historic districts that are either permitted work, or posting Stop Work orders and pursuing violation cases and possibly criminal cases on unpermitted construction/ alterations/ demolition. I love historic preservation and architectural history but being one of the few women inspectors is difficult for many reasons. I am looking to try to switch gears/careers and possibly merge interests between historic preservation and earth data science. I took the only two GIS courses while at undergrad at Bryn Mawr College and looking to delve into that mindset more! Also have never taken an online course (graduated early in Dec. 2019) so I am new to this format of learning!

**Comment by WillToon** [comment]
Hello Everyone! My name is Will and I hold a graduate certificate in GIS from the University of Denver, in addition to an MSc in Violence, Conflict, and Development from the School of Oriental and African Studies (SOAS), University of London, and a bachelor’s degree from Western Washington University in Geography & Environmental Studies with a minor in GIS. Currently, I am a young professional looking to begin my career in Geographic Information Systems and analysis with a government agency, multilateral institution or NGO focusing on environmental conservation, post-conflict reconstruction, or international development. Most recently I have been interning with the United Nations Office on Drugs and Crime in Vienna, Austria where I realized that coding and Python automation are essential to success in modern GIS careers. I'm looking forward to improving our coding skills together and learning more about earth data analytics!

**Comment by brglea** [comment]
Hi everyone! I had to start a new account so I think my previous comment was deleted. Anyways, my name is Lauren. I am currently the (only) Landmark Preservation inspector for Denver. I exclusively do construction inspections on individual landmarks or properties in historic districts. This includes permitted work and posting Stop Work Orders and pursuing violation cases and potentially criminal cases on unpermitted alterations/construction/ demolition. It has been difficult as one of the few female inspectors for many reasons and I am looking to shift gears/career to something involving earth data science. My only previous experience is 2 GIS courses at Bryn Mawr College (B.A. 20' magna cum laude) however we didn't learn coding. So I suspect I will need a lot of help and support as this is far outside my wheelhouse of historic preservation. A dream career would be to somehow merge earth data science and historic preservation but I'm not sure what that looks like!

**Reply by allenmoench** [reply]
Hi Lauren! Hmm, merging earth data and historic preservation. This has me pondering about ways to do that, since history and Anthropology are some things that fascinate me! Would be interesting to see a map of archaeological sites, and maybe correlate that with age for a particular ancient culture. Not sure beyond that at the moment, but your post got me thinking!

**Comment by allierosel** [comment]
Hi All! I am Allie a current fourth-year undergrad at CU in geology and GIS. I don't have much experience working in this field but am looking forward to finding a job or internship after I graduate. I have a little bit of experience with programming so I am excited to apply it to Earth Analytics!

**Comment by atticusbaker** [comment]
Hello! I am Atticus, in my Junior year at CU for Geophysics and GIS. I am looking forward to better my skills in coding and data science, this melds really well with my Internship as a data manager at NOAA! Looking forward to working with you all.
